data/json/items/comestibles.json

```diff
@@ -20259,5 +20261,70 @@
             "fruit": "mushroom_poison",
             "grow" : 91
         }
-}
+},{
+        "type" : "COMESTIBLE",
+        "id" : "ink_sac",
+        "name" : "ink sac",
+        "weight" : 250,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You pop the sac's membrane and smear yourself in it's contents.",
+            "effects" : [
+                {
+                "id": "inked",
+                "duration": 900
+                }
+            ]
+        },
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "MED",
+        "symbol" : "o",
+        "quench" : 0,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 0,
+        "description" : "A translucent organic sac full of black ink. It is known that cephalapod ink has scent masking properties, perhaps you should put this knowledge to use?",
+        "price" : 500,
+        "material" : "null",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : 0
+},{
+        "type" : "COMESTIBLE",
```

Comment:
You're not using this item here.
It should probably be added with later PR, not here.